Formula/mpw.rb

```diff
@@ -23,26 +23,174 @@ class Mpw < Formula
     sha256 cellar: :any, high_sierra:   "07b89df8d96f9c1cebbf6296a4e98b2bac833c45f736b646a1eba24bd5244732"
   end
 
+  depends_on "jq" => :build
   depends_on "json-c"
   depends_on "libsodium"
   depends_on "ncurses"
 
   def install
+    cd "cli" unless build.head?
     cd "platform-independent/c/cli" if build.head?
 
     ENV["targets"] = "mpw"
-    # not compatible with json-c 0.14 yet
-    ENV["mpw_json"] = "0"
+    ENV["mpw_json"] = "1"
     ENV["mpw_color"] = "1"
 
     system "./build"
-    system "./mpw-cli-tests"
     bin.install "mpw"
   end
 
   test do
-    assert_equal "Jejr5[RepuSosp",
-      shell_output("#{bin}/mpw -q -Fnone -u 'Robert Lee Mitchell' -M 'banana colored duckling' " \
-                   "-tlong -c1 -a3 'masterpasswordapp.com'").strip
```

Comment:
Why do we need to replace this test?

Also, your replacement test has many style errors. The errors start [here](https://github.com/Homebrew/homebrew-core/pull/71428/checks?check_run_id=1929426297#step:5:17) and go on for a bit.

Reply:
>_< – I forgot to run `brew audit --strict` before committing. My bad.

> Why do we need to replace this test?
We want to make sure that the host machine is handling each of the processes correctly.  This verifies that the architecture/OS/build dependencies are all handling the case soundly.  Things such as endianness, unicode, char signedness, etc should all be confirmed against the target machine (which is not the same as the build machine).

Reply:
Formula test blocks aren't the place for that sort of verification. @SMillerDev also says this here: https://github.com/Homebrew/homebrew-core/pull/71428#discussion_r578593710

The test block is just a sanity check that the build didn't go horribly wrong, and is meant to simulate a user interacting briefly with the software's functionality.

Reply:
mkay — so homebrew doesn't provide any way to allow the user to verify that the software operates reliably in their environment?

I do run integration tests during development, but this only tests the software against the development machine, not the target architecture.  Would you recommend I add a formula flag to build & install the test suite (ie. `mpw-tests`)?

Reply:
> mkay — so homebrew doesn't provide any way to allow the user to verify that the software operates reliably in their environment?

They can run it to see if it works. Anything else is up to upstream in combination with the simple test in brew.

> I do run integration tests during development, but this only tests the software against the development machine, not the target architecture.

The only advice I can give is: add more CI machines to test for macOS as well.

> Would you recommend I add a formula flag to build & install the test suite (ie. mpw-tests)?

No, we don't allow options in homebrew-core.

Reply:
You can set up GitHub Actions running macOS now. That should provide you with at least one option for running your integration tests on macOS.

GitHub runners also come with `brew` installed, so you can even run your integration tests against the bottles we build.